corehq/apps/users/views/mobile/users.py

```diff
@@ -207,6 +218,12 @@ def commtrack_form(self):
     @property
     def page_context(self):
         from corehq.apps.users.views.mobile import GroupsListView
+
+        if self.request.is_view_only:
+            make_form_readonly(self.commtrack_form)
+            make_form_readonly(self.form_user_update.user_form)
+            make_form_readonly(self.form_user_update.custom_data.form)
```

Comment:
@biyeun any reason this didn't include the `group_form`?

Reply:
@mkangia this was a year ago, so my memory isn't super crisp...but I believe we deliberately did not include groups in this scope as there were still a lot of unanswered questions as to handle groups and locations, etc.